doc/pandas-2.0/source/removals.rst

```diff
@@ -0,0 +1,78 @@
+.. _removals:
+
+================================
+ Code to remove and other ideas
+================================
+
+Dropping Python 2 support
+=========================
+
+With Python 2.7 reaching its supported end-of-life in 2020, like some other
+Python projects (e.g. IPython / Jupyter) we should seriously contemplate making
+pandas 2.0 only support Python 3.5 and higher. In addition to lowering the
+development burden at both the C API and pure Python level, we can also finally
+look to take advantage of features (things like ``asyncio``, maybe?) only
+available in Python 3.
+
+Deprecated code to remove
+=========================
+
+* ``.ix`` indexing entirely
+* ``Panel`` and ``PanelND`` classes
+* Plotting?
+
+Other ideas
+===========
+
+Here's a collection of other miscellaneous ideas that don't necessarily fit
+elsewhere in these documents.
+
+Column statistics
+~~~~~~~~~~~~~~~~~
+
+In quite a few pandas algorithms, there are characteristics of the data that
```

Comment:
yeah, I'll add that here (since we're already computing this in indexes) -- more expensive than the others since you have to push the data through hash table